NetCDF2zarrPhysModel.py

```diff
@@ -0,0 +1,207 @@
+#------------------------------------------------------------------------------------------------------------------
+# Convertion from netcdf to zarr_store using xarray and pandas libraries
+# --------------------------------------------------------------------------------------------------------------------
+#
+#   USAGE
+#
+#  python3 NetCDF2zarrPhysModel.py start_time number_batches_append batch_size
+#  or
+#  python3 NetCDF2zarrPhysModel.py --basedir base_dir_name  start_time number_batches_append batch_size
+#
+#  start_time            - begining time stamp given by user
+#  number_batches_append - how many time steps to keep
+#  batch_size            - size of time in seconds
+#
+#  start_time is provided as a string in format "dd/mm/yy HH:MM:SS"
+#
+#-----------------------------------------------------------------------------------------------------------------------'''
+
+import argparse
+from dask.distributed import Client, progress, LocalCluster
+import pandas as pd
+from pathlib import Path
+import xarray as xr
+import numpy as np
+import numcodecs
+import zarr
+from datetime import datetime
+import time
+import sys
+
+# Define time string formats:
+format_short = "ddMMyyyyHHmmSS"
+# format for time string extended
+format_extended = "dd/MM/yy HH:mm:SS"
```

Comment:
We should use ISO formatted dates here. See comment on the download script above.